src/store/actions/postActions.js

```diff
@@ -1,154 +1,188 @@
-                dispatch({ type: 'REMOVE_SUCCESS', id: post.id });
-                return 
-            }).catch((err) => {
-                dispatch({ type: 'REMOVE_ERROR', err });
-            })
-        } else {
-            dispatch({ type: 'REMOVE_ERROR', err: "Not authorized." });
-        }
+export const loadPosts = (amount) => {
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+    dispatch({ type: "LOAD_LOADING" });
+
+    let posts = firestore.collection("posts").where("visible", "==", true);
+
+    if (amount < 0) {
+      amount = 0;
     }
-}
 
-export const loadPosts = (amount) => {
-    return (dispatch, getState, {getFirebase, getFirestore}) => {
-        const firebase = getFirebase();
-        const firestore = getFirestore();
-        dispatch({ type: 'LOAD_LOADING' });
-
-        let posts = firestore.collection('posts').where("visible","==",true);
-
-        if(amount < 0){
-            amount = 0;
-        }
-        
-        posts.orderBy('timestamp','desc').limit(amount).get().then((querySnapshot) => {
-            let results = [];
-            querySnapshot.forEach(function(doc) {
-                let data = doc.data();
-                results.push({id: doc.id, data});
-            });
-            dispatch({ type: 'LOAD_SUCCESS', results });
-        })
-        .catch((err) => {
-            dispatch({ type: 'LOAD_ERROR', err });
+    posts
+      .orderBy("timestamp", "desc")
+      .limit(amount)
+      .get()
+      .then((querySnapshot) => {
+        let results = [];
+        querySnapshot.forEach(function (doc) {
+          let data = doc.data();
+          results.push({ id: doc.id, data });
         });
-    }
-}
+        dispatch({ type: "LOAD_SUCCESS", results });
+      })
+      .catch((err) => {
+        dispatch({ type: "LOAD_ERROR", err });
+      });
+  };
+};
 
 export const loadAllPosts = (amount) => {
-    return (dispatch, getState, {getFirebase, getFirestore}) => {
-        const firebase = getFirebase();
-        const firestore = getFirestore();
-        dispatch({ type: 'LOAD_LOADING' });
-
-        let posts = firestore.collection('posts');
-
-        if(amount < 0){
-            amount = 0;
-        }
-        
-        posts.orderBy('timestamp','desc').limit(amount).get().then((querySnapshot) => {
-            let results = [];
-            querySnapshot.forEach(function(doc) {
-                let data = doc.data();
-                results.push({id: doc.id, data});
-            });
-            dispatch({ type: 'LOAD_SUCCESS', results });
-        })
-        .catch((err) => {
-            dispatch({ type: 'LOAD_ERROR', err });
-        });
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+    dispatch({ type: "LOAD_LOADING" });
+
+    let posts = firestore.collection("posts");
```

Comment:
```suggestion
    let posts = firestore.collection("posts");
 
    dispatch({ type: "LOAD_LOADING" });

   
```
Is that possible? If so, adjust this. If not check this area for newlines.